examples/react/src/index.css

```diff
@@ -11,13 +11,8 @@
 
 body {
   width: 100%;
-  height: 100vh;
+  height: 90vh;
   margin: 0;
   padding: 0;
   max-height: 100lvh;
-}
-
-#root {
-  width: 100%;
-  height: 100%;
-}
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 12 to 24]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->You should not remove those, I think your issues comes from the fact that you didn't provide a container for the iframe<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/cohort-xyz/cohort-embed-js/1?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->